bee-node/bee-node/Cargo.toml

```diff
@@ -37,24 +41,20 @@ fxhash = { version = "0.2.1", default-features = false }
 hex = { version = "0.4.3", default-features = false }
 iota-crypto = { version = "0.9.1", default-features = false, features = [ "ed25519", "random", "blake2b" ] }
 log = { version = "0.4.14", default-features = false }
-mime_guess = { version = "2.0.3", default-features = false, optional = true }
 multiaddr = { version = "0.13.0", default-features = false }
 paho-mqtt = { version = "0.9.1", default-features = false, features = [ "bundled" ] }
```

Comment:
I guess this one is not needed anymore ?
Maybe do a pass to check if there are more unused deps.

Reply:
No, it was only used in the dashboard plugin, so it's now a dependency of `bee-plugin-dashboard` instead. `cargo udeps` doesn't report anything.

Reply:
@Adam-Gleave You resolved but you didn't remove it, maybe I misunderstood something ?

Reply:
Oh, just me being stupid -- I saw `mime_guess` highlighted and assumed you were asking about why it was gone.